nixos/modules/services/networking/nomad.nix

```diff
@@ -27,6 +27,17 @@ in
         '';
       };
 
+      enableDocker = mkOption {
+        type = types.bool;
+        default = true;
```

Comment:
I'm not sure this should be a configuration option, since there's more than just the docker daemon running need to launch containers with nomad (https://www.nomadproject.io/docs/drivers/docker#client-requirements). You have to enable the plugin on clients.

Additionally, even if we keep this I think it should default to `false`, because there's no reason to enable docker on nomad servers, just clients since servers aren't going to launch containers.

Reply:
Discussed in https://github.com/NixOS/nixpkgs/pull/105739#discussion_r559244603